webapp/src/components/modals/channel_subscriptions/__snapshots__/edit_channel_subscription.test.tsx.snap

```diff
@@ -3630,6 +3630,17 @@ exports[`components/EditChannelSubscription should match snapshot after fetching
              that since you have not selected a security level filter, the subscription will only allow issues that have no security level assigned.
           </span>
         </div>
+        <div
+          className="channel-subscriptions-modal__learnMore"
+        >
+          <a
+            href="https://github.com/mattermost/mattermost-plugin-jira?tab=readme-ov-file#create-a-channel-subscription"
```

Comment:
The link seems broken. It does take me to the top of the page, not a specific section.

Reply:
@hanzei I think that is because the PR has not been merged yet. The link will take to a specific section after merging the PR.

Reply:
Thanks for pointing that out; I missed that the section is now.

Still, I wonder about the `tab` parameter. Can we simplify to

```suggestion
            href="https://github.com/mattermost/mattermost-plugin-jira#create-a-channel-subscription"
```
?